scss/_patterns_buttons.scss

```diff
@@ -136,6 +137,33 @@ $hover-background-opacity-percentage: $hover-background-opacity-amount * 100%;
   }
 }
 
+@mixin vf-button-link {
+  .p-button--link {
+    @extend %vf-link-base;
+
+    border: none;
+    border-radius: 0;
+    margin-right: 0;
+    margin-top: 0;
```

Comment:
I believe his overrides another 0 - is it really necessary?

Reply:
Ah yes - updated!